templates/besu.service.j2

```diff
@@ -6,11 +6,11 @@ After=syslog.target network.target
 User={{ besu_user }}
 Group={{ besu_group }}
 Environment=HOME=/home/{{ besu_user }}
-{% if besu_log4j_config_file %}
+{% if besu_log4j_config_file != "" %}
 Environment=LOG4J_CONFIGURATION_FILE={{ besu_log4j_config_file }}
 {% endif %}
 {% if besu_env_opts %}
-Environment='BESU_OPTS={{ besu_env_opts|besu_opts_format }}'
+Environment='BESU_OPTS={{besu_env_opts|map('to_json')|join(' ')}}'
```

Comment:
I assume this is a breaking change, correct?

That is, if I've set `besu_env_opts='"-Xmx8G" "-Xms2G"'` or something like that, will I need to change this to a list format for it to work properly?